### ISSUE TITLE
Backend error when trying to create a kiezkassen proposal

### ISSUE THREAD

**Issue by xi** [liqd/adhocracy3.mercator#1119]
When trying to create a kiezkassen proposal I get the following error from the backend (in a batch request):

```json
{
    "code": 400,
    "body": {
        "status": "error",
        "errors": [{
            "location": "body",
            "name": "content_type",
            "description": "\"<InterfaceClass adhocracy_meinberlin.resources.kiezkassen.IProposalVersion>\" is not one of "
        }]
    }
}
```

**Comment by xi** [comment]
Something similar happens when trying to create a mercator proposal:

```
"<InterfaceClass adhocracy_mercator.resources.mercator.IMercatorProposal>" is not one of <InterfaceClass adhocracy_core.resources.asset.IPoolWithAssets>, <InterfaceClass adhocracy_core.resources.external_resource.IExternalResource>, <InterfaceClass adhocracy_core.resources.pool.IBasicPool>
```

This really blocks my work!

**Comment by MagdaN** [comment]
I have the same issue and cannot go on

**Comment by pallix** [comment]
fixed by #1120

**Comment by pallix** [comment]
For Mercator the permissions are set in `resource/subscriber.py` and reflect the current status of the platform i.e it's not possible to create proposal anymore now.
